webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const webpack = require('webpack');
+require('dotenv').config();
+
+const SRC_DIR = path.join(__dirname, '/client/src');
+const DIST_DIR = path.join(__dirname, '/client/dist');
+
+module.exports = {
```

Comment:
Missing a resolve for jsx, js, etc file endings per Airbnb style guide

Reply:
Added resolve for .jsx, .jsx, and .json to webpack config.